src/detect-engine-threshold.h

```diff
@@ -30,6 +30,14 @@
 #include "ippair.h"
 #include "host-storage.h"
 
+typedef enum SigThresholdResults_ {
+    THRESHOLD_DONT_ALERT = 0,     // Rule doesn't match
+    THRESHOLD_ALERT = 1,          // Rule match
+    THRESHOLD_SILENT_MATCH = 2,   // Rule match, alert is suppressed, rule actions apply
+    THRESHOLD_SUPPRESSED = 0,     // Alert suppressed due to thresholding
+    THRESHOLD_NOT_SUPPRESSED = 1, // Rule match, alert not suppressed
+} SigThresholdResults;
```

Comment:
Adding comments has helped in combing through the variants, but not sure if now they're good enough.

Reply:
would like to see this split into 2 enums. One for the top 3 values, another for the bottom 2